src/css/elements/_typography.scss

```diff
@@ -16,11 +16,30 @@ h6 {
   font-weight: 700;
 }
 
-h1 { font-size: 3rem; }
-h2 { font-size: 2.375rem; }
-h3 { font-size: 1.875rem; }
-h4 { font-size: 1.25rem; }
-h5 { font-size: 1rem; }
+h1,
+.h1-size {
+  font-size: 3rem;
```

Comment:
whoah I've got some learning to do. this frontend stuff gets crazy real fast. Here I was thinking we just had `%` `px` and `em` and maybe another, but now you're throwing American rock bands at me? Idk if I am cut out for this stuff. 

:laughing:

Reply:
🎸 👨‍🎤 👩‍🎤 🎸